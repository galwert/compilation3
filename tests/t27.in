void printOK() {
    print("OK!");
    printi(5);
}

void main()
{
    int x = (((((5)))));
    byte Bro = 5 b; // really <Partner>?

    x = (((((x + Bro)+x)+x)));
    y = (((((((((x+y))))))+1))+3);
}