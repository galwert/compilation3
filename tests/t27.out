---end scope---
line 12: variable y is not defined
